app/events/zync_event.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class ZyncEvent < RailsEventStore::Event
+class ZyncEvent < BaseEventStoreEvent
```

Comment:
Why using this one?

Reply:
OK right it is the only event model that did not inherit from the BaseEventStoreEVent